client.go

```diff
@@ -555,6 +554,11 @@ func (c *TunaSessionClient) DialWithConfig(remoteAddr string, config *nkn.DialCo
 		return nil, err
 	}
 
+	remoteNknAddr, err := c.multiClient.ResolveDest(remoteAddr)
```

Comment:
Since we don't need original address, the easiest way to avoid mistake (forgot to change `remoteAddr` to `remoteNknAddr` somewhere now or in the future) is to overwrite `remoteAddr` variable with
`remoteAddr, err = ...`
Same for UDP

Reply:
Yes, modified.